api/client/service/update.go

```diff
@@ -33,6 +34,11 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringSlice("command", []string{}, "Service command")
 	flags.StringSlice("arg", []string{}, "Service command args")
 	addServiceFlags(cmd, opts)
+	flags.StringSlice(flagEnvRemove, []string{}, "Remove an environment variable")
+	flags.StringSlice(flagLabelRemove, []string{}, "The key of a label to remove")
```

Comment:
I like the help text for removing environment variables above. "Remove an environment variable" puts the word "remove" front and center and uses an active voice that makes the description clearer. I'd suggest rephrasing this one as something like "Remove a label by its key", and so on for the others.